### PR TITLE
[ALLUXIO-2065] Use constants in FileSystemStatisticsTest

### PR DESCRIPTION
**Pull request by xxhuang**
Use constants in FileSystemStatisticsTest

https://alluxio.atlassian.net/browse/ALLUXIO-2065

### CODE REVIEW

**Comment by ifcharming** [comment]
ok to test

File: tests/src/test/java/alluxio/hadoop/FileSystemStatisticsTest.java [hunk 1]
```diff
@@ -34,9 +34,10 @@
 
   private static final int BLOCK_SIZE = 128;
   private static final int FILE_LEN = BLOCK_SIZE * 2 + 1;
+  private static final int WORKER_CAPACITY_BYTES = 10000;
   @ClassRule
   public static LocalAlluxioClusterResource sLocalAlluxioClusterResource =
-      new LocalAlluxioClusterResource(10000, BLOCK_SIZE);
+      new LocalAlluxioClusterResource(WORKER_CAPACITY_BYTES, BLOCK_SIZE);
```
Comment (ifcharming) [inline tests/src/test/java/alluxio/hadoop/FileSystemStatisticsTest.java]: instead of `10000`, can you please use 10 * `Constants.KB` ?

**Comment by ifcharming** [comment]
@xxhuang thank you for the contribution to Alluxio!
The PR looks great, I've left a minor comment.

File: tests/src/test/java/alluxio/hadoop/FileSystemStatisticsTest.java (hunk 1 above)
Comment (xxhuang) [inline tests/src/test/java/alluxio/hadoop/FileSystemStatisticsTest.java]: hm...It looks like there is no int KB in Constants.class. And Constants.class cannot be changed(in a .jar file).  Do I need to import other library for Constant.class? Or should I just create an  Contans_KB = 1000 in FileSystemStatisticsTest and use it?@ifcharming

Comment (ifcharming) [inline tests/src/test/java/alluxio/hadoop/FileSystemStatisticsTest.java]: KB exists in `alluxio.Constants`
Can you add `import alluxio.Constants;` ?

Comment (xxhuang) [inline tests/src/test/java/alluxio/hadoop/FileSystemStatisticsTest.java]: Oops...Found it. My bad...Thanks for your information! @ifcharming

**Comment by apc999** [comment]
LGTM

**Pull request merged**